pkg/controller/compliancescan/scan.go

```diff
@@ -235,6 +246,58 @@ func (r *ReconcileComplianceScan) deleteScanPods(instance *compv1alpha1.Complian
 	return nil
 }
 
+func (r *ReconcileComplianceScan) addTailoringVolume(instance *compv1alpha1.ComplianceScan, pod *corev1.Pod) error {
+	mode := int32(0744)
```

Comment:
Does the tailoring file need to be executable?

Reply:
Good point, let me change that.